app/src/main/kotlin/co/touchlab/droidconandroid/ScheduleActivity.kt

```diff
@@ -247,6 +266,9 @@ class ScheduleActivity : AppCompatActivity(), ConferenceDataViewModel.Host {
         private val POSITION_EXPLORE = 1
         private val POSITION_MY_SCHEDULE = 2
         private val ALL_EVENTS = "all_events"
+        private val ALL_TOPIC = "all_debug"
```

Comment:
probably shouldnt have the debug postfix anymore. But perhaps we should do "all_2017" so that we can test without sending notifications out to 2016 users